src/routes/index.js

```diff
@@ -1,11 +1,22 @@
-var express = require("express");
-var router = express.Router();
+const express = require("express");
+const router = express.Router();
 require("dotenv").config();
 const axios = require("axios");
 const background = process.env.BACKGROUND_COLOR;
 
+function getBaseUrl() {
+  let url = process.env.API_BASE_URL;
+
+  if(!/^https:\/\//.test(url)) {
+    url = url.replace('http://', ''); // remove unsecured protocol, if exists
+    url = `https://${url}`;
+  }
```

Comment:
```suggestion
  const urlContainsScheme = /^https?:\/\//.test(url);
  if(!urlContainsScheme) {
    url = `https://${url}`;
  }
```

Better if we only add `https;//` when there's no scheme provided at all. Also adding extra line to help with readability.